MacPass/MPPathControl.m

```diff
@@ -39,11 +39,10 @@ - (void)showOpenPanel:(id)sender {
   if([self.delegate respondsToSelector:@selector(pathControl:willDisplayOpenPanel:)]) {
     [self.delegate pathControl:self willDisplayOpenPanel:panel];
   }
-  [panel beginWithCompletionHandler:^(NSModalResponse result) {
-    if(result == NSModalResponseOK) {
-      self.URL = panel.URLs.firstObject;
-    }
-  }];
+  NSModalResponse result = [panel runModal];
```

Comment:
Nice improvement, I would go even further and use sheets - how about:
```
  [panel beginSheetModalForWindow:self.window completionHandler:^(NSModalResponse result) {
    if(result == NSModalResponseOK) {
      self.URL = panel.URLs.firstObject;
    }
  }];

```

Reply:
Done